addons/survey/controllers/survey_session_manage.py

```diff
@@ -110,14 +111,32 @@ def survey_session_next_question(self, survey_token, go_back=False, **kwargs):
             })
             request.env['bus.bus'].sendone(survey.access_token, {
                 'question_start': now.timestamp(),
-                'type': 'next_question'
+                'type': 'next_question',
+                'go_back': go_back,
             })
 
             template_values = self._prepare_manage_session_values(survey)
             template_values['is_rpc_call'] = True
-            return request.env.ref('survey.user_input_session_manage_content')._render(template_values)
+
+            # get the question that follows the one to return (for background management)
+            most_voted_answers = survey._get_session_most_voted_answers()
+            following_question = survey._get_next_page_or_question(most_voted_answers, next_question.id)
+
+            background_image_url = next_question.background_image_url
+            reload_background_on_next = survey.has_conditional_questions or next_question.background_image_url != following_question.background_image_url
+
+            template_values.update({
+                'background_image_url': background_image_url,
+                'reload_background_on_next': reload_background_on_next,
+            })
+
+            return {
+                'background_image_url': background_image_url,
+                # 'reload_background_on_next': reload_background_on_next,
+                'question_html': request.env.ref('survey.user_input_session_manage_content')._render(template_values)
+            }
```

Comment:
As this is only done here (when AJAX submitting the question), and not in the "survey_session_manage" route, I'm wondering if it ill work if the survey host makes a refresh of the page (F5) mid-session?